org-jira.el

```diff
@@ -349,6 +349,15 @@ Entry to this mode calls the value of `org-jira-mode-hook'."
   "Return the components the ISSUE belongs to."
   (mapconcat (lambda (comp) (org-jira-find-value comp 'name)) (org-jira-find-value issue 'fields 'components) ", "))
 
+(defun org-jira-insert (text &optional coding-system)
```

Comment:
Do you think this would work better by setting the coding-system in a buffer level org-jira-coding-system variable, and changing org-jira-insert to work similar to insert, with a `&rest args` style parameter list, so multiple strings could be inserted without explicitly calling concat on them?